answers/Maker.md

```diff
@@ -0,0 +1,179 @@
+
+#### Risk Governance Framework
+
+To protect itself from bias, polarizing debate, and self serving rhetoric, Maker has created an internal process that utilizes an objective "risk governance framework". In this process, "risk teams" (professionals employed by the Maker Foundation) utilize a formal, rigorous framework for continually evaluating the qualitative and quantitative risks associated with various collateral types. For instance the volatility risk, liquidity risk, and stability of the asset fundamentals. The outputs of this framework are then input into well-understood risk models borrowed from traditional finance to determine optimal "risk parameters" such as the debt ceiling, liquidation ratio, stability fee, and other parameters. The core team then presents their new models, data and suggested parameters to MKR holders and the community at large. Feedback from the community is incorporated and then put into a proposal for an executive vote. The executive vote itself can be used to gather further feedback from MKR holders, which can be incorporated back into the proposal. Eventually, MakerDAO intends to further decentralize this process, creating multiple risk teams elected by MKR holders that compete with each other using the risk governance framework, creating a "decentralized, open scientific risk management community".
+
+#### Community engagement
+
+Maker incorporates input into governance decisions from the community at large via active engagement on the its [chat](https://chat.makerdao.com/), subreddit ([r/MakerDAO](https://www.reddit.com/r/MakerDAO)), GitHub, and other platforms. Weekly governance and risk meetings are live streamed, recordings of which are hosted on [YouTube](https://www.youtube.com/playlist?list=PLLzkWCj8ywWNq5-90-Id6VPSsrk4OWVan) and [Soundcloud](https://soundcloud.com/makerdao/sets/governance-and-risk). Meeting minutes of meetings are hosted on [github](https://github.com/scottrepreneur/maker-minutes). When larger scale changes to the system are considered, on-chain "governance votes" are held to poll community sentiment before implementing them.
+
+A [Governance Dashboard](https://vote.makerdao.com/) displays governance-related information and real-time voting data .
+
+
+# Blockchain Governance
+
+On-chain voting is enabled using the MKR "governance" token. To vote on proposed changes to the system, MKR holders must lock their MKR in a "voting contract", a smart contract on the Ethereum blockchain which will automatically enact the proposed changes to the Maker system if the vote passes. 1 MKR = 1 vote. Voting contracts can propose a single change or multiple changes at once.
```

Comment:
> lock their MKR
What does this mean? time-lock? unable to change vote once it is cast?

Reply:
Was able to get more details via the Maker chat. Updated wording:

> On-chain voting is enabled using the MKR "governance" token. To vote on proposed changes to the system, MKR holders must lock their MKR in a "voting contract" in return for IOU tokens. 1 IOU token = 1 vote. Voting consists of sending these IOU tokens to a "proposal contract" that represents a proposed change of state to the system. Proposal contracts can propose a single change, or multiple changes at once. Voters can also vote for up to five proposals at once. If voting for multiple proposals, they all receive equal weighting. Voters can change their votes at any time.